src/graphics/aurora/model_kotor.cpp

```diff
@@ -1122,6 +1121,24 @@ void ModelNode_KotOR::readSaber(Model_KotOR::ParserContext &ctx) {
 	std::memcpy(indexData, indices, 96 * sizeof(uint16));
 }
 
+void ModelNode_KotOR::readEmitter(Model_KotOR::ParserContext &ctx) {
+	// TODO: Add values for unknown fields
+	ctx.mdl->skip(20); // Unknown
+
+	uint32 xGrid = ctx.mdl->readUint32LE();
+	uint32 yGrid = ctx.mdl->readUint32LE();
+
+	ctx.mdl->skip(4); // Unknown
+
+	// TODO: The padding bytes are not necessarily zero, maybe they are specific parameters for the specified mode?
```

Comment:
Could also be that the game just doesn't initialize the padding and writes garbage